Assets/MixedRealityToolkit.Services/InputSystem/PointerBehavior.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    /// <summary>
+    /// Specifies how a pointer in MRTK's default input system behaves.
+    /// On: Pointer is always on, regardless of what other pointers are active.
+    /// Off: Pointer is always off, regardless of what other pointers are active.
+    /// Default: Pointer active state is managed by MRTK input system. If it is a near pointer (grab, poke), it
+    /// will be always enabled. If it is not a near pointer, it will get disabled if any near pointer on the 
+    /// same hand is active. This is what allows rays to turn off when a hand is near a grabbable.
+    /// 
+    /// By default, pointer behaviors are set to default state.
+    /// </summary>
+    public enum PointerBehavior
+    {
+        /// <summary>
+        /// Pointer is always on, regardless of what other pointers are active.
+        /// </summary>
+        On,
+        /// <summary>
+        /// Pointer is always off, regardless of what other pointers are active.
+        /// </summary>
+        Off,
+        /// <summary>
+        /// Pointer active state is managed by MRTK input system. If it is a near pointer (grab, poke), it
+        /// will be always enabled. If it is not a near pointer, it will get disabled if any near pointer on the 
+        /// same hand is active. This is what allows rays to turn off when a hand is near a grabbable.
+        /// </summary>
+        Default
```

Comment:
Nit: Move the Default to location zero, so that in the case that things get initialized to zero, the default is actually chosen as the default, not on.

Otherwise I think if this gets persisted and defaults get chosen things will just be "AlwaysOn"